admin/includes/languages/lang.english.php

```diff
@@ -583,6 +583,10 @@
     'TEXT_SALEMAKER_IMMEDIATELY' => 'Immediately',
     'TEXT_SALEMAKER_NEVER' => 'Never',
     'TEXT_SET_DEFAULT' => 'Set as default',
+    'TEXT_SHIPPING_lbs' => '(lbs)',
```

Comment:
Should these be UPPER case
`'TEXT_SHIPPING_LBS' => '(lbs)',`
etc

Reply:
See PR #6245